### DISCUSSION TITLE
Recreating example from the tailwindcss website

### DISCUSSION THREAD

**Discussion by Alexander-Trinh** [tailwindlabs/tailwindcss#5491] [Help]
So I just discovered tailwind css and I like how modular it is. To learn I am trying to play with the examples provided on the tailwindcss landing page. Specifically the brutalist style card with the "Retro Shoe".

So in the playground I copied the code from the website and also took the config file from the tailwindcss.com github page. However I cant seem to get the same effect as it is shown on the website. Here is my attempt:
https://play.tailwindcss.com/qMBJvU2HaD

Could someone tell me what Tailwind CSS classes am I missing to get the exact effect as on the website (specifically the size radio buttons and the shadow on the buy now and add to tag buttons).

Thanks!

**Comment by simonswiss** [comment]
Hey!

That's a pretty good start 🎉

You can inspect the example directly on the website to see how it's done.

I am sure you could create custom drop shadows for this brutalist style, but here instead it is using an absolutely positioned rectangle positioned _behind_ the button, and slightly offset with `-bottom-0.5` and `left-0.5`:

![image](https://user-images.githubusercontent.com/485747/133548268-5cf58ae0-eef8-40d3-bc6f-6f2011911b27.png)

I've updated your Play with the fake shadows:

https://play.tailwindcss.com/oRoHOuiwOk

![image](https://user-images.githubusercontent.com/485747/133548212-57a691e8-f772-4cf8-ade6-a9a9f1ad6ef9.png)

Same deal for the radios – it's an absolutely position "shape":

![image](https://user-images.githubusercontent.com/485747/133548334-de37fe50-063a-4b06-b056-4d8111b0a188.png)
